### DISCUSSION TITLE
How to integrate v3 designer in Angular

### DISCUSSION THREAD

**Discussion by NrujalSharma** [elsa-workflows/elsa-core#3813] [General]
I integrated v2 designer following [this](https://jeremiedevillard.wordpress.com/2021/09/14/using-elsa-workflow-dashboard-in-angular-app/) demo by @jdevillard .

v3 designer seems to be different as the folders mentioned in above tutorial are no longer available in the v3 branch. I couldn't find any resources to integrate v3 with Angular.

How to integrate v3 designer with my existing Angular application?

**Comment by jdevillard** [comment]
Hello , 

Ive done the integration with v3 . I hope PR will be discuss this week to be available quickly .

**Reply by NrujalSharma** [reply]
Hey @jdevillard ,
Thank you!

**Reply by amTrinh** [reply]
could you please give me more information about your PR?
